staging/dex-k8s-authenticator/html-templates/linux-mac-common.html

```diff
@@ -8,7 +8,7 @@ <h3>Copy IDP CA Certificate From URL</h3>
       <button class="btn" style="float:right" data-clipboard-snippet="">
         <img class="clippy" width="13" src="{{ .Web_Path_Prefix }}static/clippy.svg" alt=""/>
       </button>
-      <pre><code>curl --create-dirs -s {{ .IDPCaURI }} -o ${HOME}/.kube/certs/{{ .ClusterName }}/idp-ca.crt</code></pre>
+      <pre><code>curl --create-dirs -s {{ .IDPCaURI }} -o ${HOME}/.kube/certs/{{ .Username }}-{{ .ClusterName }}/idp-ca.crt</code></pre>
```

Comment:
question: where is the Username coming from ?

Reply:
Would that work when attaching existing Kubernetes clusters like GKE and deploying addons there ?